### ISSUE TITLE
Code doesn't correspond to the description in the paper.

### ISSUE THREAD

**Issue by pangzss** [renwang435/CYBORGS#1]
Hi, thank you for sharing the code of your great work.

The code seems not to correspond to the implementation details in the paper.
For example, the code has no warm-up stage, and the vmf_loss_weight is set to 0 by default.
The current algorithm flow doesn't align with Algorithm 3 in the supplementary material.

Moreover, the code fails to run because the dataloader is initialized when masks have not been generated.
Other hyperparameters, such as batch size and start learning rate, also differ between code and paper.

Could you upload the code for producing the results in the paper? Or could you provide some advice on the reproduction of your results in the paper based on the current code? 

Thanks a lot.
